extensions/sql-migration/src/telemtery.ts

```diff
@@ -32,7 +33,8 @@ export enum TelemetryAction {
 	StartMigration = 'StartMigration',
 	CutoverMigration = 'CutoverMigration',
 	CancelMigration = 'CancelMigration',
-	MigrationStatus = 'MigrationStatus'
+	MigrationStatus = 'MigrationStatus',
+	PageButtonAction = 'PageButtonAction'
```

Comment:
This is already an action event, this value should be describing what the action is. So something like `PageButtonClick` would be clearer.